### PR TITLE
feat: Initial implementation of importAccountsByEncryptedUrl on MNW and Meteor Wallet

### PR DESCRIPTION
**Pull request by hcho112**
# Description

This PR contains initial implementation for adding `importAccountsByEncryptedUrl` interface for `My Near Wallet` and `Meteor Wallet`. [WEP-211](https://pagodaplatform.atlassian.net/browse/WEP-211) 

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# Type of change.

- [ ] FIX - a PR of this type patches a bug.
- [x] FEATURE - a PR of this type introduces a new feature.
- [ ] BUILD - a PR of this type introduces build changes.
- [ ] CI - a PR of this type introduces CI changes.
- [ ] DOCS - a PR of this type introduces DOCS improvement.
- [ ] STYLE - a PR of this type introduces style changes.
- [ ] REFACTOR - a PR of this type introduces refactoring.
- [ ] PERFORMANCE - a PR of this type introduces performance changes.
- [ ] TEST - a PR of this type adds more tests.
- [ ] CHORE - a PR introduces other changes than the specified above.

### CODE REVIEW

**Comment by andy-haynes** [comment]
@hcho112 you beat me to the punch 😅 

Would you mind reviewing #575 and rebasing this on top once you're satisfied please? I'll update the document to reflect the changes I've made there

**Comment by hcho112** [comment]
@andy-haynes I have commented Meteor wallet's `buildImportAccountsUrl` until we know for sure the wallet type of the Meteor Wallet is `BrowserWallet` type.

File: packages/my-near-wallet/src/lib/my-near-wallet.spec.ts [hunk 1]
```diff
@@ -111,3 +111,18 @@ describe("signAndSendTransaction", () => {
     expect(result).toEqual(null);
   });
 });
+
+describe("buildImportAccountsUrl", () => {
+  it("returns import url", async () => {
+    const { wallet } = await createMyNearWallet();
+
+    expect(wallet.buildImportAccountsUrl).toBeDefined();
```
Comment (andy-haynes) [inline packages/my-near-wallet/src/lib/my-near-wallet.spec.ts:119]: Could we expect that this is a function instead please?

File: packages/my-near-wallet/src/lib/my-near-wallet.spec.ts [hunk 2]
```diff
@@ -111,3 +111,18 @@ describe("signAndSendTransaction", () => {
...
+
+    const url =
+      wallet.buildImportAccountsUrl && wallet.buildImportAccountsUrl();
+
+    expect(`${url}#test`).toEqual(
+      "https://testnet.mynearwallet.com/batch-import#test"
+    );
```
Comment (andy-haynes) [inline packages/my-near-wallet/src/lib/my-near-wallet.spec.ts:126]: Let's simplify this to just
```js
expect(wallet.buildImportAccountsUrl()).toEqual('https://testnet.mynearwallet.com/batch-import');
```

File: packages/meteor-wallet/src/lib/meteor-wallet.spec.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @nrwl/nx/enforce-module-boundaries */
```
Comment (andy-haynes) [inline packages/meteor-wallet/src/lib/meteor-wallet.spec.ts:1]: Let's just shelve this until we have Meteor sorted out. No value in adding an empty test at the moment

File: packages/meteor-wallet/src/lib/meteor-wallet.ts [hunk 4]
```diff
@@ -195,6 +195,11 @@ const createMeteorWalletInjected: WalletBehaviourFactory<
         transactions: await transformTransactions(transactions),
       });
     },
+
```
Comment (andy-haynes) [inline packages/meteor-wallet/src/lib/meteor-wallet.ts:198]: Same as above, let's shelve until we have resolution on Meteor

**Review by andy-haynes — CHANGES_REQUESTED** [review]

**Review by andy-haynes — APPROVED** [review]

**Pull request merged**